docs/changes/cosmology/11639.api.rst

```diff
@@ -0,0 +1,2 @@
+Allow ``m_nu`` to be input as any quantity-like, and converted to eV with a
```

Comment:
It was already true that it was converted with mass-energy enabled. What's also new is that now floats are allowed.